src/cgame/rocket/rocket_documents.cpp

```diff
@@ -59,19 +58,7 @@ void Rocket_LoadDocument( const char *path )
 
 void Rocket_LoadCursor( const char *path )
 {
-	Rocket::Core::ElementDocument* document = menuContext->LoadMouseCursor( path );
-
-	if( document )
-	{
-		// This gets 12px on 1920×1080 screen, which is libRocket default for 1em
-		int fontSize = std::min(cgs.glconfig.vidWidth, cgs.glconfig.vidHeight) / 90;
-
-		// 1.6×2.3em ≈ 20×28px on 1920×1080 screen
-		document->SetProperty( "width", va( "%fpx", 1.6 * fontSize ) );
-		document->SetProperty( "height", va( "%fpx", 2.3 * fontSize ) );
-
-		document->RemoveReference();
-	}
+	// TODO: Remove.
```

Comment:
I guess this TODO is done in later commits? Can't really assert anything here, but maybe it would be possible to use [deprecated](https://en.cppreference.com/w/cpp/language/attributes/deprecated) attribute to avoid forgetting (would have a compiler warning, which can be checked or turned as error)?
Since C++14 is required for RMLUI anyway, let's use it?